pkg/registry/common/refresh/nse_registry_client_test.go

```diff
@@ -70,12 +70,30 @@ func (c *checkExpirationTimeClient) Unregister(ctx context.Context, in *registry
 	return new(empty.Empty), nil
 }
 
+func TestIsStable_TestNewNetworkServiceEndpointRegistryClient(t *testing.T) {
+	for i := 0; i < 100; i++ {
+		TestNewNetworkServiceEndpointRegistryClient(t)
+	}
+}
+
+func TestIsStable_TestNewNetworkServiceEndpointRegistryClient_CalledRegisterTwice(t *testing.T) {
+	for i := 0; i < 100; i++ {
+		TestNewNetworkServiceEndpointRegistryClient_CalledRegisterTwice(t)
+	}
+}
+
+func TestIsStable_TestRefreshNSEClient_ShouldSetExpirationTime_BeforeCallNext(t *testing.T) {
+	for i := 0; i < 100; i++ {
+		TestRefreshNSEClient_ShouldSetExpirationTime_BeforeCallNext(t)
+	}
+}
```

Comment:
Result for these tests https://github.com/networkservicemesh/sdk/pull/438/commits/d01f2205799b90393daa15158b9a42766d1a4ed4